onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -9,6 +9,7 @@
 import sys
 import threading
 import unittest
+from pathlib import Path
```

Comment:
Same import modules only goes here

Reply:
For the why:

Import only the module instead to keep the name space clean and provide readers with more context on where its members come from. For example, readers don’t need to backtrack to see `sleep` is a function from `time`, as opposed to a function defined in the file. https://google.github.io/styleguide/pyguide.html#22-imports